HC2025: Hero, Events, Recap Mobile

geoff-jiang opened the pull request:
<!--- Add a GIF that describes how you feel about this PR (or just a cool one) -->
![](https://media1.tenor.com/m/RjINDshJG48AAAAd/cat-smile.gif)

## Description
<!--- Describe your changes! Include screenshots/video if applicable -->
Yeah, i did some stuff. Ignore the tug of war animation I thought it was based on scroll and I started doing some Math LOL


## Other considerations
<!--- Workarounds, planned future changes, special notes, etc. -->

geoff-jiang committed:
feat: tried to implement tugofwar animation, added first mobile hero section

geoff-jiang committed:
finished hero page + learn section, no navbar

geoff-jiang committed:
added events section, modified sign component

geoff-jiang committed:
feat: mobile recap section

geoff-jiang committed:
feat: navbar and fixes

geoff-jiang committed:
fix: linting errors, comment out tugofwar animation

tdanielles commented:
looking good, tyyy! 🔥 

a couple things:
1. Events and recap are both gone on desktop TT (not sure if this was on purpose / u commented it out, or if u just commented out the tug of war animation)
<img width="1461" height="818" alt="Screen Shot 2025-09-25 at 11 07 47 PM" src="https://github.com/user-attachments/assets/80a10f87-525d-44d2-a50d-7ea855f2a702" />
2. Need more padding/margin at the bottom of the menu bar; should be balanced with top space
<img width="946" height="2046" alt="image" src="https://github.com/user-attachments/assets/74366cf0-7ef8-437f-84b4-8890199afe11" />
3. might be better if font size is a taaad bit smaller for title, desc, and buttons in hero
<img width="946" height="2046" alt="image" src="https://github.com/user-attachments/assets/eaa2924f-b098-448e-a94d-1dc13443a591" />
4. Recap title not centered, video not fitted, the house(?) layout isn't centered either; images r also blurry
<img width="946" height="2046" alt="image" src="https://github.com/user-attachments/assets/fedd1594-13b7-45e9-a5d5-129b38cc39f0" />

geoff-jiang committed:
fix: layout and image issues on mobile

geoff-jiang commented:
@tdanielles 
  1. Commented sections out so it wouldn’t look weird during testing. I think Kez changed this section anyways so we’ll have to work out how it’ll merge 
  2. Fixed
  3. Fixed to match design
  4. Fixed

daniel-panhead commented:
<img width="471" height="828" alt="image" src="https://github.com/user-attachments/assets/b0b795bd-cac4-4225-af45-c628823150c8" />

Sign still looks weird to me - is the layout scaling properly with screen width?

<img width="496" height="819" alt="image" src="https://github.com/user-attachments/assets/a3c56613-9a63-40aa-ac2e-958aab0d336b" />

Menu entries are overflowing past menu bar

Rest looks good though!

geoff-jiang committed:
fix: mobile responsivness for text

geoff-jiang committed:
Merge branch 'hackcamp2025_dev' into geoff/mobile

geoff-jiang committed:
fix: changed sign to include text
